mesh/ncmesh.cpp

```diff
@@ -771,6 +809,30 @@ void NCMesh::CheckAnisoPrism(int vn1, int vn2, int vn3, int vn4,
    }
 }
 
+void NCMesh::CheckAnisoPyramid(int vn1, int vn2, int vn3, int vn4,
```

Comment:
I'm not sure we need this duplicate of `NCMesh::CheckAnisoPrism` (and related block in `CheckAnisoFace`), as aniso refinement for pyramids is not supported, so propagation of forced refinements should not be of concern. If pyramids are present in the mesh, aniso refinement should be disabled, since we can't guarantee the mesh will stay OK once the aniso refinements start propagating.

Reply:
I think those are probably left over from when I was trying to define aniostropic refinements on the pyramids.  I'll disable them and test everything out to make sure I didn't miss anything.

Reply:
Removed